### DISCUSSION TITLE
Question about listed refs when importing subtrees from a private repository.

### DISCUSSION THREAD

**Discussion by 3213-cmd** [magit/magit#5034] [Support] [answered]
I have looked troughout the issues and tried to look a little bit into the code, but I am not very good with Emacs Lisp.
I am also not too familliar with what is appropriate for issues, so I hope it is okay to ask here. Since I could not find any help online.

When adding a subtree using magit is it normal that at the step **"determining available refs"** if the repository that I am adding as a subtree is private (requires credentials) that no refs will be shown?  When I try to add a public repository I can see all the refs. 
I am also aware that I can fill in the ref myself even though it's not listed. So it is nothing urgent to me.

I am just asking because I believe I have seen this step "determining available refs" at other times and want to know if that is intended behavior.

Thank you very much for your time in advance.

**Comment by tarsius** [comment, marked as answer]
Based on a *quick* look at the code, I don't think whether the other repository is public or private is what makes the difference, but whether you have already added that as a remote beforehand and fetched it.
